docs/understanding-the-flutterflow-ui/projects/how-to-create-find-organize-projects.md

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
Can we move this page into a different sidebar location? "FlutterFlow Resources" or something like that? I dont think this section belongs in the "Understanding the FlutterFlow UI" section

Reply:
Yep. This was all done before we had figured out that "Resources was deleted"